packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -1818,11 +1818,33 @@ class BackdropFilterLayer extends ContainerLayer {
     }
   }
 
+  /// The blend mode to use to apply the filtered background content onto the background
```

Comment:
(I can't comment on the line, but): Update the constructor above to also allow configuring this?

Reply:
This is probably the source of a number of discrepancies here. The constructor here is not used to set any values in practice. Instead, the layer is constructed with no arguments and then the fields are set directly on the constructed values. Other levels of the framework work on a different scheme where either an object is instantiated or updated depending on the history of the object and so the properties are never "temporarily null"/

I'll try to make everything internally consistent here with the way it is actually used.

Reply:
As a result of this distinction about the way that layers are constructed and maintained, the same language won't work here for discussing the issues about the chosen mode. The docs here become more of a warning about which mode should be used for compatibility than a description of a local default.

Reply:
The constructor was updated as appropriate and the logic was changed to default the value when it is passed on rather than on the property itself.